src/tools/createMonitoringGoals.js

```diff
@@ -0,0 +1,154 @@
+import {
+  sequelize,
+  GoalTemplate,
+} from '../models';
+import { auditLogger } from '../logger';
+
+const createMonitoringGoals = async () => {
+  const cutOffDate = '2024-10-01';
+  const monitoringTemplateName = '(Monitoring) The recipient will develop and implement a QIP/CAP to address monitoring findings.';
```

Comment:
This is a bad design to have the goals text hard coded.

Reply:
updated.

Reply:
That will fail in testing.

I have set you the following a few times now on how to get a dynamically populated unique value to use. It would need to be placed in a migration file and then the alias could be used to find them distinctly regardless of the id.

```
ALTER TABLE "GoalTemplates"
ADD COLUMN standard TEXT GENERATED ALWAYS AS (
    CASE
        WHEN "creationMethod" = 'Curated' THEN substring("templateName" from '(?:^[(]([^)]+)[)])')
        ELSE NULL
    END
) STORED;

CREATE UNIQUE INDEX unique_standard_non_null
ON "GoalTemplates"(standard)
WHERE standard IS NOT NULL;
```

Then add the new column to the model. It will be populated only on the curated goals and be the value within the ()

Reply:
I'm not sure I fully understand why the other way wouldn't have worked. I have added the suggested migration to create the 'standard' column and updated the code to use it.